docs/chrraces.md

```diff
@@ -10,23 +10,26 @@ This dbc contains all possible races, some of which are unused and unavailable t
-| 13     | CinematicSequence | Integer      | id from CinematicSequence.dbc                   |
-| 14     | unk_322           |              | faction (0 Alliance, 1 Horde, 2 not available?) |
-| 15-30  | name              | String + Loc | used for DBC language detection/selection       |
-| 31     |                   |              | string flags, unused                            |
-| 32-47  | nameFemale        | String + Loc | if different from base (male) case              |
-| 48     |                   |              | string flags, unused                            |
-| 49-64  |                   |              | if different from base (male) case              |
-| 65     |                   |              | string flags, unused                            |
-| 66-68  |                   |              | unused                                          |
-| 69     | expansion         | Integer      | (0 - original race, 1 - tbc addon, ...)         |
+| 3      | FactionID         | Integer      | Faction ID                                      |
+| 4      | ExplorationSoundID| Integer      |                                                 |
+| 5      | model_m           | Integer      | Male display ID, used for character creation    |
+| 6      | model_f           | Integer      | Female display ID, used for character creation  |
+| 7      | ClientPrefix      | String       |                                                 |
+| 8      | TeamID            | Integer      | 7=Alliance, 1=Horde                             |
+| 9      | CreatureType      | Integer      | 7=Humanoid                                      |
+| 10     | ResSicknessSpellID| Integer      | Spell to use for res sickness                   |
+| 11     | SplashSoundID     | Integer      | Sound to use for splashing                      |
+| 12     | ClientFilestring  | String       |                                                 |
+| 13     | CinematicSequence | Integer      | ID from CinematicSequence.dbc                   |
+| 14     | Alliance          | Integer      | Faction (0=Alliance, 1=Horde, 2=Not available)  |
+| 15-30  | name              | String + Loc | Used for DBC language detection/selection       |
+| 31     | NameLangMask      | Integer      | String flags, unused                            |
+| 32-47  | nameFemale        | String + Loc | If different from base case, otherwise unused   |
+| 48     | fNameLangMask     | Integer      | String flags, unused                            |
+| 49-64  | nameMale          | String + Loc | If different from base case, otherwise unused   |
+| 65     | mNameLangMask     | Integer      | String flags, unused                            |
+| 66-68  | CustomizationNames| String       | Strings to use for customization in creation    |
+| 69     | expansion         | Integer      | 0=Original race, 1=TBC race                     |
```

Comment:
Update to use filed/type/note to match https://wowdev.wiki/DB/ChrRaces#Wrath

Reply:
There are a couple of the Lang masks at the bottom that the page didn't have listed, so those were kept. Otherwise, should be the same.